migrations/0002_create-profile.sql

```diff
@@ -0,0 +1,67 @@
+-- Migration number: 0002 	 2025-01-30T00:58:21.685Z
+
+DROP TABLE IF EXISTS profile;
+
+-- A person's profile inside the database
+-- It should contain no sensitive information
+CREATE TABLE IF NOT EXISTS profile (
+	-- The UUID, stored as text
+	id TEXT NOT NULL UNIQUE COLLATE BINARY,
+	-- The person's email,
+	-- It is used as a reference for the user
+	-- And should be unique inside the system
+	email TEXT NOT NULL UNIQUE,
+	-- Schema version to use
+	schemaVersion INTEGER NOT NULL DEFAULT 1,
+	-- The person's name, or how they like to be identified
+	name TEXT NOT NULL,
+	-- A text blurb the person can provide about themselves
+	description TEXT,
+	-- A flag indicating if the user is based on the Grater Toronto Area (GTA)
+	isBasedOnGTA INTEGER NOT NULL DEFAULT 1 CHECK(isBasedOnGTA IN (0, 1)),
+	-- A flag indicating if the user is available to join local/in-person events
+	canJoinLocalEvents INTEGER NOT NULL DEFAULT 1 CHECK(canJoinLocalEvents IN (0, 1)),
+	-- The pronouns the person identifies with
+	pronouns TEXT,
+	-- The person's birthday, saved as month and day only
+	-- In the format: YYYY-MM
```

Comment:
The format appears to be year and month instead of month and day as you suggest on line 26. Should this then be:
`MM-DD` ?

Reply:
Good catch! It was supposed to be month and day, I mistiped it.